README.md

```diff
@@ -70,6 +70,23 @@ directory the variables from `.env` will be loaded into the environment.
 
 E.g. `foreman run rails server` or `foreman run rails console`.
 
+## Quick Start  
+###Beginner Friendly Tips for New Contributors  
+- After forking the repo, follow the steps above (Bootstrap (& optionally Mailtrap))
+- (Install and) connect to Postgres server  
+- With everything properly installed, open the browser (in development environment)
```

Comment:
@F3PiX Ik zou hier de haakjes rondom "in development environment" weghalen. Je gebruikt al best veel haakjes en deze zijn niet echt nodig denk ik. Maar dat zijn details :)